jdaviz/configs/default/plugins/plot_options/plot_options.py

```diff
@@ -242,12 +301,61 @@ def line_visible(state):
                                                   'line_as_steps_value', 'line_as_steps_sync')
         self.uncertainty_visible = PlotOptionsSyncState(self, self.viewer, self.layer, 'show_uncertainty',  # noqa
                                                         'uncertainty_visible_value', 'uncertainty_visible_sync')  # noqa
 
+        # Scatter/marker options:
+        self.marker_visible = PlotOptionsSyncState(self, self.viewer, self.layer, 'visible',
+                                                   'marker_visible_value', 'marker_visible_sync',
+                                                   state_filter=is_scatter)
+        self.marker_fill = PlotOptionsSyncState(self, self.viewer, self.layer, 'fill',
+                                                'marker_fill_value', 'marker_fill_sync',
+                                                state_filter=is_scatter)
+        self.marker_opacity = PlotOptionsSyncState(self, self.viewer, self.layer, 'alpha',
+                                                   'marker_opacity_value', 'marker_opacity_sync',
+                                                   state_filter=is_scatter)
+        self.marker_size_mode = PlotOptionsSyncState(self, self.viewer, self.layer, 'size_mode',
+                                                     'marker_size_mode_value', 'marker_size_mode_sync',  # noqa
+                                                     state_filter=is_scatter)
+        self.marker_size = PlotOptionsSyncState(self, self.viewer, self.layer, 'size',
+                                                'marker_size_value', 'marker_size_sync',
+                                                state_filter=is_scatter)
+        self.marker_size_scale = PlotOptionsSyncState(self, self.viewer, self.layer, 'size_scaling',
+                                                      'marker_size_scale_value', 'marker_size_scale_sync',  # noqa
+                                                      state_filter=is_scatter)
+        self.marker_size_col = PlotOptionsSyncState(self, self.viewer, self.layer, 'size_att',
+                                                    'marker_size_col_value', 'marker_size_col_sync',
+                                                    state_filter=is_scatter)
+        self.marker_size_vmin = PlotOptionsSyncState(self, self.viewer, self.layer, 'size_vmin',
+                                                     'marker_size_vmin_value', 'marker_size_vmin_sync',  # noqa
+                                                     state_filter=is_scatter)
+        self.marker_size_vmax = PlotOptionsSyncState(self, self.viewer, self.layer, 'size_vmax',
+                                                     'marker_size_vmax_value', 'marker_size_vmax_sync',  # noqa
+                                                     state_filter=is_scatter)
+
+        # TODO: remove marker_ prefix if these also apply to the lines?

```

Comment:
Even if we don't wait for the updates to glue-jupyter to support lines in scatter viewers, we might want to at least understand the behavior and what we want these to be called for the user-api.  Ultimately jdaviz doesn't use any of these, or expose any of these to the user, so making incremental changes to these wouldn't be a problem (just will need matching updates downstream to `lcviz`).